week-5-practise/js/main.js

```diff
@@ -0,0 +1,68 @@
+var blueButton = document.getElementById("blueBtn");
+blueButton.addEventListener("click", addClick);
```

Comment:
You had the right idea with buttonClick.
```suggestion
blueButton.addEventListener("click", () => buttonClick(`#588fbd`, `#ffa500`, "black", "white"));
```